### PR TITLE
Update attachment upload screen.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

File: shared/chat/conversation/attachment-get-titles/index.tsx [hunk 1]
```diff
@@ -96,89 +83,174 @@ class GetTitles extends React.Component<Props, State> {
...
+              autoCorrect={true}
+              placeholder={titleHint}
+              multiline={true}
+              value={info.title}
+              onEnterKeyDown={this._onNext}
+              onChangeText={this._updateTitle}
+              selectTextOnFocus={true}
+            />
+          </Kb.Box2>
+          <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.buttonContainer}>
```
Comment (buoyad) [inline shared/chat/conversation/attachment-get-titles/index.tsx]: I think this should be `Kb.ButtonBar`

File: shared/chat/conversation/attachment-get-titles/index.tsx [hunk 2]
```diff
@@ -96,89 +83,174 @@ class GetTitles extends React.Component<Props, State> {
...
 
     return (
-      <Kb.ScrollView style={{height: '100%', width: '100%'}}>
-        <Kb.Box style={Styles.isMobile ? stylesMobile : stylesDesktop}>
-          <Kb.Box style={{...Styles.globalStyles.flexBoxCenter, height: 150, width: 150}}>
-            {info.type === 'image' ? (
-              <Kb.OrientedImage
-                src={Styles.isAndroid ? `file://${path}` : path}
-                style={Styles.isMobile ? {height: 150, width: 150} : {maxHeight: '100%', maxWidth: '100%'}}
+      <Kb.ScrollView style={styles.scrollView}>
```
Comment (buoyad) [inline shared/chat/conversation/attachment-get-titles/index.tsx]: :sheep: on iOS this extends down behind the keyboard, so you can't get to the button without blurring the input (on the 5S sim). Would be nice for this to stay in the `KeyboardAvoidingView`

File: shared/chat/conversation/attachment-get-titles/index.tsx (hunk 1 above)
Comment (adamjspooner) [reply shared/chat/conversation/attachment-get-titles/index.tsx]: It was, but I kept running into issues with it and just went with `Box2` instead. I'll try `ButtonBar` again, though.

File: shared/chat/conversation/attachment-get-titles/index.tsx (hunk 2 above)
Comment (adamjspooner) [reply shared/chat/conversation/attachment-get-titles/index.tsx]: Interesting. I was testing on the 5s emulator and never ran into this. I'll take a look.

**Comment by adamjspooner** [comment]
@buoyad, mind taking another look?

**Pull request merged**